doc/README

```diff
@@ -1,27 +0,0 @@
-
```

Comment:
The reason for removing this file?

Reply:
Since I intended to give a general overview in the main README, I considered having a second README in a subfolder redundand(*). But maybe I should leave it lying around as a reminder until the content has been integrated elsewhere and not just remove it. I'll revert that removal for the moment.

(*) For example the main README already contains a [Manual Pages](https://github.com/mspncp/openssl/blob/revamp-the-docs/README.md#manual-pages) section. Currently I only link to the official manpages, but I could also add a reference (link) to to the manual sources.

Reply:
Removal of doc/README has been reverted in 5d855f6e20ea791e4a296f2cad69bc7bb9f21f0c.